### PR TITLE
add coin-iotx

### PR DESCRIPTION
**Pull request by metapolymer**
----

*Checklist:*

- README.md includes:
  - [x] The details of signing data and transaction data, this includes parameters.
  - [x] Official docs or white paper.
  - [x] Website or api can query assets and broadcast transaction.

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by kanitsu** [comment]
Please resolve the conflicts.
Also, please remove unused variables/import and format the codes according to [.prettierrc](https://github.com/metapolymer/coolwallet-sdk/blob/iotex/.prettierrc)
Thank you.

**Comment by kanitsu** [comment]
Thanks for your update.
It seems that payload argument is missing in StakeCreate, StakeUnstake, and StakeWithdraw.
Since they're available in the Antenna SDK, please add them for better future compatibility.
Thank you.

**Comment by kanitsu** [comment]
Please also add:
1. XRC-20 token transfer
   -Need separate signing method, since we also want to show token symbol on the card.
2. Stacking's add deposit
    -Please refer to Envelop.stakeAddDeposit

File: packages/coin-tester/src/components/coins/iotx/index.tsx [hunk 1]
```diff
@@ -0,0 +1,777 @@
...
+
+  const intToHex = (i: Integer, padBytes = 0) => bnToHex(new BigNumber(i), padBytes);
+
+  const prepareXrc20Token = async () => {
+    handleState(async () => {
+      const appId = localStorage.getItem('appId');
+      if (!appId) throw new Error('No Appid stored, please register!');
+      let [nonce, gasLimit, gasPrice, amount, recipient, tokenDecimals, tokenSymbol, tokenAddress] = xrc20TokenArgs;
+      const tokenAmount = new BigNumber(amount).shiftedBy(parseInt(tokenDecimals, 10)).toFixed();
+      const data = 'a9059cbb000000000000000000000000' + decodeAddr(recipient)
```
Comment (kanitsu) [inline packages/coin-tester/src/components/coins/iotx/index.tsx:340]: Why don't use encodeXRC20TokenInfo in utils?

**Review by kanitsu — APPROVED** [review]

**Review by skchang0710 — APPROVED** [review]

**Pull request merged**